android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -308,6 +312,37 @@ class OkHttpTest {
     assertEquals(sessionIds[0], sessionIds[1])
   }
 
+  @Test
+  fun testDnsOverHttps() {
+    assumeNetwork()
+
+    client = client.newBuilder()
+        .eventListenerFactory(LoggingEventListener.Factory())
+        .build()
+
+    val dohDns = buildCloudflareIp(client)
+    val dohEnabledClient =
+        client.newBuilder().eventListenerFactory(EventListener.NONE.asFactory()).dns(dohDns).build()
```

Comment:
curious – why NONE ?

Reply:
```
    client = client.newBuilder()
        .eventListenerFactory(LoggingEventListener.Factory())
        .build()
```

I just wanted this test to log requests for DNS, not the overall HTTP request.